server/src/middlewares/errorHandler.js

```diff
@@ -16,6 +16,6 @@ export default class ErrorHandler {
     if (res.headersSent) {
       return next(err);
     }
-    res.status(err.status || 500).json({ error: err.message });
+    return res.status(err.status || 500).json({ error: err.message });
```

Comment:
good job remembering to add `return` so that you don't get `headers already sent` errors.

Reply:
thanks...i found it through my test coverage